mfp_functions/apiStatusCheck.js

```diff
@@ -7,46 +7,46 @@ var fetchSingleDate = require('./fetchSingleDate');
 //this function checks to see if MyFitnessPal has changed the structure of their data presentation
 //by checking to see if running 'fetchSingleDate' returns the expected data values for a known page
 
-var apiStatusCheck = function(callback){
+var apiStatusCheck = function (callback) {
```

Comment:
Not a great idea to submit PRs with these types of style / spacing changes, since it breaks the consistency of the codebase.

It doesn't have any functional implications, and mfp is way overdue for an overhaul anyways, so I won't make a big deal of it. But I would suggest avoiding committing such changes on public projects that aren't your own.

I have since switched to this preferred spacing: `function(arg) {...body}`, but I'll throw Airbnb linter and prettier into this project and that will take care of everything all in one go.